data/json/items/armor/torso_armor.json

```diff
@@ -36,7 +36,45 @@
     "looks_like": "armor_cuirass",
     "name": { "str": "XS bell cuirass", "str_pl": "XS bell cuirasses" },
     "proportional": { "weight": 0.75, "volume": 0.75, "price": 0.5 },
-    "flags": [ "VARSIZE", "OUTER", "STURDY", "UNDERSIZE" ]
+    "extend": { "flags": [ "UNDERSIZE" ] }
+  },
+  {
+    "id": "cuirass_bronze",
```

Comment:
I would say this item could be a smidge better if it requires all the same proficiencies and books other armors do, and 10hrs to construct.

It's less thick and covering than the bell cuirass so perhaps its encumbrance could be dropped somewhat to compensate.

Reply:
Hmmm, alright, I can adjust the item and the recipe a bit.
The bell cuirass currently has 7 encumbrace when fitted, so I could make this item have 6, as I don't want to make it too low.
And I could also shave off another hour from the recipe, as it's currently only an hour shorter than the bell.
Do you think that's a good way to go about it?

Reply:
I didn't realize the bell could be fitted... I guess my point is, it's not like this item is a poorly made armor. It takes hours, fabrication, book references, and forge tools so if its got worse coverage, worse protection and worse encumbrance over the bell cuirass, whats the point? I don't think it's imbalanced, or wrong as is but I don't know if it's really filling any niche.

Reply:
I planned for it to be a basic piece of armor for survivors that can't find an armorsmithing book, because the recipe is auto-learned at fabrication 5.
The book references are there to give the player the recipe for it if their skill is at 4, but it should be auto-learned at 5.
That gives bronze use in runs with no access to books, or at least that's what I was planning it to do.

Though I do agree the recipe should be tweaked, I am just unsure how exactly.
As is, it's basically just copy-pasted recipe for the bell cuirass with a few tweaks, such as taking slightly less bronze to craft.

Reply:
ohhhh I didn't see the autolearn, my bad. In that case I think this is good as is 👍 

might be worth adding a sentence to the description saying something like "this isn't particularly well designed" so players can at a glance see that they would want the bell cuirass if they have the books.